File: Kotlin/BubbleSort.kt
fun bSort(arr:IntArray):IntArray{
    var swap = true
    while(swap){
        swap = false
        for(i in 0 until arr.size-1){
            if(arr[i] > arr[i+1]){
                val temp = arr[i]
                arr[i] = arr[i+1]
                arr[i + 1] = temp
                swap = true
            }
        }
    }
    return arr
}


fun main(args: Array<String>) {
    val list = bSort(intArrayOf(2,12,3,9,4))
    for (i in list) print("$i ")
}

File: C/BubbleSort.c
#include<stdio.h>
int main()
{
    int n;
    scanf("%d",&n);
    int a[n];
    for (int k = 0; k < n; k++)
    {
        scanf("%d",&a[k]);
    }
    for (int i = 0; i < n-1; i++)
    {
        for (int j = 0; j < n-i-1; j++)
        {
            if(a[j]>a[j+1]){
                int temp=a[j];
                a[j]=a[j+1];
                a[j+1]=temp;
            }
        }
    }
    for (int m = 0; m < n; m++)
    {
        printf("%d ",a[m]);
    }
    return 0;
}